docs/self-managed/platform-deployment/helm-kubernetes/deploy.md

```diff
@@ -220,9 +220,9 @@ If the output of the `describe` command was not beneficial, tail the logs of the
 kubectl logs -f <POD_NAME>
 ```
 
-## Upgrading Camunda Helm chart
+## Updating Camunda Helm chart
 
-For upgrading Camunda Helm chart from one release to another, visit the [Camunda Platform 8 Helm upgrading page](upgrade.md).
+For updating Camunda Helm chart from one release to another, visit the [Camunda Platform 8 Helm updating page](update.md).
 
```

Comment:
https://helm.sh/docs/helm/helm_upgrade/

`helm update` is the command you issue and not something within our control. For SEO purposes, we should include both "update" and "upgrade" in the page or page metadata.

You may want to say "For updating Camunda Helm chart from one release to another, you'll need to perform a `helm upgrade`". We can look for other examples too since this is a bit confusing.

Reply:
This makes sense to me and is consistent with your other comments about referring to what the command calls. Adjusting in latest commit 👍